### ISSUE TITLE
Why does the loss function subtract the Frobenius norm of S?

### ISSUE THREAD

**Issue by ashawkey** [jiangboahu/GLCN-tf#2]
https://github.com/jiangboahu/GLCN-tf/blob/master/glcn/models.py#L73
Why this line substract the loss?
Also, `losslr2` is set to a rather small value, is this item really matters in the performance of the model?

**Comment by ashawkey** [comment]
Also I went into problem when training on citeseer dataest, the loss turned negative and behaved strangely. Is this caused by the subtraction too?

`
Epoch: 0485 train_loss= -50.20491 train_acc= 0.97500 val_loss= -52.41239 val_acc= 0.68000 test_acc= 0.68600 time= 0.14273
Epoch: 0486 train_loss= -34.68251 train_acc= 0.97500 val_loss= -52.80213 val_acc= 0.67800 test_acc= 0.68600 time= 0.14251
Epoch: 0487 train_loss= -28.91805 train_acc= 0.95000 val_loss= -53.21044 val_acc= 0.67800 test_acc= 0.68700 time= 0.14800
Epoch: 0488 train_loss= -50.68908 train_acc= 0.98333 val_loss= -53.63916 val_acc= 0.67600 test_acc= 0.68800 time= 0.15157
Epoch: 0489 train_loss= -39.81816 train_acc= 0.96667 val_loss= -54.07666 val_acc= 0.67600 test_acc= 0.68800 time= 0.14384
Epoch: 0490 train_loss= -43.86405 train_acc= 0.98333 val_loss= -54.49774 val_acc= 0.67600 test_acc= 0.68900 time= 0.14693
Epoch: 0491 train_loss= -40.59904 train_acc= 0.95833 val_loss= -54.92601 val_acc= 0.67400 test_acc= 0.68900 time= 0.14257
Epoch: 0492 train_loss= -41.85332 train_acc= 0.95000 val_loss= -55.35519 val_acc= 0.67400 test_acc= 0.68600 time= 0.14169
Epoch: 0493 train_loss= -16.11497 train_acc= 0.96667 val_loss= -55.75022 val_acc= 0.67400 test_acc= 0.68500 time= 0.14708
Epoch: 0494 train_loss= -30.45110 train_acc= 0.96667 val_loss= -56.13781 val_acc= 0.67400 test_acc= 0.68400 time= 0.14491
Epoch: 0495 train_loss= -47.51535 train_acc= 0.95000 val_loss= -56.53138 val_acc= 0.67200 test_acc= 0.68100 time= 0.13695
Epoch: 0496 train_loss= -21.71214 train_acc= 0.95000 val_loss= -56.93042 val_acc= 0.67000 test_acc= 0.67900 time= 0.13712
Epoch: 0497 train_loss= -35.81029 train_acc= 0.94167 val_loss= -57.31550 val_acc= 0.66800 test_acc= 0.68000 time= 0.14254
Epoch: 0498 train_loss= -50.04920 train_acc= 0.95833 val_loss= -57.71409 val_acc= 0.66800 test_acc= 0.67900 time= 0.14096
`
